specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -3956,6 +4152,329 @@
       },
       "description": "Response from a List Indexes request. If successful, it includes the full definitions of all indexes."
     },
+	"Skillset": {
+		"properties": {
+			"name": {
+				"externalDocs": {
```

Comment:
I think this `externalDocs` should be on the `Skillset` type, not the `name` property.

Reply:
Thanks. Moved it out on Skillset.